web/src/stream_data.ts

```diff
@@ -646,6 +646,11 @@ export function get_stream_privacy_policy(stream_id: number): string {
     return settings_config.stream_privacy_policy_values.private_with_public_history.code;
 }
 
+export function is_archived(stream_id: number): boolean {
+    const sub = sub_store.get(stream_id);
+    return sub ? sub.archived : false;
+}
```

Comment:
This file expects 100% test coverage because it only contains easily tested logic, you should just include a test for the new function rather than adding the file to an exclude list.

Reply:
Fixed.